2021-09-21 20:05:29,970 INFO    MainThread:32199 [wandb_setup.py:_flush():69] setting env: {}
2021-09-21 20:05:29,970 INFO    MainThread:32199 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-21 20:05:29,970 INFO    MainThread:32199 [wandb_init.py:_log_setup():348] Logging user logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_200529-2wg8eitb/logs/debug.log
2021-09-21 20:05:29,970 INFO    MainThread:32199 [wandb_init.py:_log_setup():349] Logging internal logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_200529-2wg8eitb/logs/debug-internal.log
2021-09-21 20:05:29,971 INFO    MainThread:32199 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f30793eb970>
2021-09-21 20:05:29,971 INFO    MainThread:32199 [wandb_init.py:init():381] calling init triggers
2021-09-21 20:05:29,972 INFO    MainThread:32199 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-21 20:05:29,972 INFO    MainThread:32199 [wandb_init.py:init():430] starting backend
2021-09-21 20:05:29,972 INFO    MainThread:32199 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-21 20:05:29,978 INFO    MainThread:32199 [backend.py:ensure_launched():135] starting backend process...
2021-09-21 20:05:29,983 INFO    MainThread:32199 [backend.py:ensure_launched():139] started backend process with pid: 32243
2021-09-21 20:05:29,985 INFO    MainThread:32199 [wandb_init.py:init():435] backend started and connected
2021-09-21 20:05:30,034 INFO    MainThread:32199 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-09-21 20:05:30,035 INFO    MainThread:32199 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-21 20:05:30,035 INFO    MainThread:32199 [wandb_init.py:init():494] updated telemetry
2021-09-21 20:05:30,036 INFO    MainThread:32199 [wandb_init.py:init():517] communicating current version
2021-09-21 20:05:30,971 INFO    MainThread:32199 [wandb_init.py:init():522] got version response 
2021-09-21 20:05:30,972 INFO    MainThread:32199 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-21 20:05:31,402 INFO    MainThread:32199 [wandb_init.py:init():557] starting run threads in backend
2021-09-21 20:05:32,503 INFO    MainThread:32199 [wandb_run.py:_console_start():1605] atexit reg
2021-09-21 20:05:32,504 INFO    MainThread:32199 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-09-21 20:05:32,504 INFO    MainThread:32199 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-09-21 20:05:32,505 INFO    MainThread:32199 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-21 20:05:32,506 INFO    MainThread:32199 [wandb_init.py:init():582] run started, returning control to user process
2021-09-21 20:05:32,511 INFO    MainThread:32199 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 20:05:32,511 INFO    MainThread:32199 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 20:05:32,729 INFO    MainThread:32199 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 20:05:32,730 INFO    MainThread:32199 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 20:05:32,730 INFO    MainThread:32199 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 20:05:32,940 INFO    MainThread:32199 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 20:05:33,647 INFO    MainThread:32199 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 20:05:33,648 INFO    MainThread:32199 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 20:05:33,848 INFO    MainThread:32199 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 20:05:33,849 INFO    MainThread:32199 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 20:05:33,849 INFO    MainThread:32199 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 20:05:34,134 INFO    MainThread:32199 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 20:05:34,142 INFO    MainThread:32199 [wandb_setup.py:_flush():69] setting env: {}
2021-09-21 20:05:34,142 INFO    MainThread:32199 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:_log_setup():348] Logging user logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_200534-1hrd4ztd/logs/debug.log
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:_log_setup():349] Logging internal logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_200534-1hrd4ztd/logs/debug-internal.log
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:init():381] calling init triggers
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 2wg8eitb
2021-09-21 20:05:34,145 INFO    MainThread:32199 [wandb_run.py:finish():1275] finishing run mlwell-rgb/gta/2wg8eitb
